docs/source/standard/util.rst

```diff
@@ -74,12 +76,13 @@ Schema
 :changes:
     List of `Change` objects
 
-
 .. _guarantee:
 
 Guarantee
 =========
 
+Can be edited during the rectificationPeriod. Value can not be greater than value.amount.
```

Comment:
Plz consider the review here openprocurement/openprocurement.auctions.insider#95